docs/content/key_combinations.md

```diff
@@ -0,0 +1,3 @@
+# Key Combinations
+
+**TODO:** Write this.
```

Comment:
Will be completed in a separate PR?

Reply:
Yes, I will work on cleaning up the doc + README next.